plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/FontUtil.java

```diff
@@ -54,4 +84,7 @@ public void widgetDisposed(DisposeEvent event) {
     });
   }
 
+  private static boolean hasStyle(FontData fontData, int style) {
+    return (fontData.getStyle() & style) != 0;
```

Comment:
javadoc for `getStyle()`:
>Returns the style of the receiver which is a bitwise OR of one or more of the SWT constants NORMAL, BOLD and ITALIC.

So I think we need the bitwise operation. What do you think?